ssts/segmentation/segmentation_gmm.py

```diff
@@ -112,20 +112,19 @@ def transform(self, data, outliers=None):
         else:
             data = data.reshape(n, c)
 
-        if outliers is not None:
-            outliers = outliers.flatten()
-
-        data = SegmenterGMM.remove_outliers(data, outliers)
```

Comment:
If we remove the outliers from the data, we won't be able to reconstruct the label matrix with dimensions (h, w). This didn't come up before because the dataset we were using didn't have outliers.